src/services/entity_builder.js

```diff
@@ -149,6 +149,23 @@ export default class EntityBuilder {
     return {...params};
   }
 
+  validateAndCastFindAssetsParams(params) {
+    const allowedParametersList = ['page', 'perPage', 'createdBy'];
+
+    validateFieldsConstrainedToSet(params, allowedParametersList);
+
+    this.ensureGeoLocationParamsCorrectlyPlaced(params);
```

Comment:
No data field in asset = no geo data field in asset.

Reply:
👍